.changeset/quiet-coins-shout.md

```diff
@@ -0,0 +1,23 @@
+---
+'@twilio-paste/box': minor
+'@twilio-paste/text': minor
+'@twilio-paste/styling-library': minor
+'@twilio-paste/style-props': minor
+'@twilio-paste/theme': patch
+'@twilio-paste/core': minor
+---
+
+[Box] Update tests to account for `stylis` parser change in jest emotion matchers.
```

Comment:
Action item (blocking): needs to be updated before this is merged.

Reply:
As general guidance we only need changelog entries for when we affect the compiled, deliverable package.

So any change to something in `src/` or the `package.json` file needs a changelog entry because the consumer of the package needs to care about it.

Anything else, stories, tests, snapshots, aren't customer deliverables, so they don't need changelog entries. It's effectively noise to the consumer at that stage as they won't be able to clearly grok what is directly going to affect them